packages/errors/src/context.ts

```diff
@@ -33,9 +35,15 @@ export type SolanaErrorContext = DefaultUnspecifiedErrorContextToUndefined<{
         addresses: string[];
         cause?: Error;
     };
+    [SOLANA_ERROR__INVALID_ADDRESS]: {
+        reason?: string;
```

Comment:
I'd be interested to hear @steveluscher's opinion on error "sub-categories" which lead us to have this optional `reason` string here.

Reply:
Let's not.

1. Reasons are not scrutable like error codes are, without matching on the _text_ of the `reason`
2. There's a chance someone will surface this message to the user (bad; English only)